packages/lisk-p2p/src/peer.ts

```diff
@@ -176,19 +186,19 @@ export class Peer {
 		return this._nodeStatus;
 	}
 
-	public set inboundSocket(value: any) {
+	public set inboundSocket(value: SCServerSocket | undefined) {
```

Comment:
why is this possible to insert `undefined`?
same for below

Reply:
Getter and setter need to have the same signature i.e. if the set value is `SCServerSocket` then get should also be `SCServerSocket` and not  `SCServerSocket | undefined` which is not possible as inbound or outbound connection could be `undefined` at some point. 

but this could be avoided if we use a regular function to set the value instead of a setter. If it looks weird then we can change it.

Reply:
created PR#988 to remove getters and that will solve this issue.